Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -3248,7 +3248,7 @@ - (id)accessibilityAttributeValue:(NSString*)attribute forParameter:(id)paramete
         return createNSArray(operationResult).autorelease();
     }
 
-    if ([attribute isEqualToString:NSAccessibilityUIElementsForSearchPredicateParameterizedAttribute]) {
+    if ([attribute isEqualToString:@"AXRangesForSearchPredicate"]) {
```

Comment:
Maybe we should define a constant for this? I know it's not done consistently now but I think it'd be a nice goal to get rid of magic strings and constants from the source.